sentry-jul/src/main/java/io/sentry/jul/SentryHandler.java

```diff
@@ -0,0 +1,283 @@
+package io.sentry.jul;
+
+import io.sentry.Breadcrumb;
+import io.sentry.Sentry;
+import io.sentry.SentryEvent;
+import io.sentry.SentryLevel;
+import io.sentry.SentryOptions;
+import io.sentry.logback.BuildConfig;
+import io.sentry.protocol.Message;
+import io.sentry.protocol.SdkVersion;
+import io.sentry.util.CollectionUtils;
+import java.text.MessageFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.ErrorManager;
+import java.util.logging.Filter;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+import java.util.logging.LogRecord;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.TestOnly;
+import org.slf4j.MDC;
+
+/** Logging handler in charge of sending the java.util.logging records to a Sentry server. */
+public final class SentryHandler extends Handler {
+  /** Name of the {@link SentryEvent} extra property containing the Thread id. */
+  public static final String THREAD_ID = "thread_id";
+  /**
+   * If true, <code>String.format()</code> is used to render parameterized log messages instead of
+   * <code>MessageFormat.format()</code>; Defaults to false.
+   */
+  private boolean printfStyle;
+
+  private @NotNull Level minimumBreadcrumbLevel = Level.INFO;
+  private @NotNull Level minimumEventLevel = Level.SEVERE;
+
+  /** Creates an instance of SentryHandler. */
+  public SentryHandler() {
+    this(new SentryOptions(), true);
+  }
+
+  /** Creates an instance of SentryHandler. */
+  @TestOnly
+  SentryHandler(final @NotNull SentryOptions options, final boolean configureFromLogManager) {
+    setFilter(new DropSentryFilter());
+    if (configureFromLogManager) {
+      retrieveProperties();
+    }
+    if (!Sentry.isEnabled()) {
+      options.setEnableExternalConfiguration(true);
+      options.setSdkVersion(createSdkVersion(options));
+      Sentry.init(options);
```

Comment:
I don't think this is going to be used by Android at all, but it'd be possible, right now we are limiting it as the Handler inits Sentry itself, another approach would be to make it an Integration, as we do with `Timber`, so people initialize either the pure Java or Android SDK.
maybe @bruno-garcia have opinions?

Reply:
is `jul` used on Android at all?

The limitations with this approach is that on Android the SDK would need to be init'ed before the logging integration which should be the case (either the AndroidManifest or initing right at the start of the app) so I think it's OK to go ahead with this.

Reply:
that's what I've tried to say, it's not common at all, but it'd be possible, JUL is part of the Android SDK, for people that care about APK size, its been always an option, it's a trade-off, I'd prefer one-liner integration and making it possible on the mobile world other than not possible at all, well, at least no device context, etc.

Reply:
On mobile, Sentry initializes before or after handler? If Sentry gets initialized after handler is created, it will overwrite what was configured in the handler.

Reply:
if `auto-init` mode is on, it's before

Reply:
as the JUL supported has been asked by Java and not Android people, let's keep it as it is and if at some point Android people ask for that, we reconsider that.